policyengine_us/variables/gov/states/vt/tax/income/credits/vt_veteran_tax_credit.py

```diff
@@ -0,0 +1,64 @@
+from policyengine_us.model_api import *
+
+
+class vt_veteran_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont veteran tax credit"
+    definition_period = YEAR
+    unit = USD
+    reference = [
+        "https://vtdigger.org/2025/06/25/gov-phil-scott-signs-13-5-million-tax-credit-package-benefiting-low-income-workers-families-retirees-and-veterans/"
+    ]
+    defined_for = StateCode.VT
+    documentation = "Vermont veteran tax credit providing $250 refundable credit for low-income veterans as part of S.51 (2025) tax relief package."
+
+    def formula(tax_unit, period, parameters):
+        # Only applies starting in 2025
+        if period.start.year < 2025:
+            return 0
```

Comment:
Still not a fan of such dependencies, the credit is only part of the computation tree in 2025. I know when unit testing we return a value but there won't be a sim where the `vt_veteran_tax_credit` is computed for any year pre-2025 unless the `refundable.yaml` parameter is adjusted. 
I would remove this as it just adds complexity